### DISCUSSION TITLE
extract error when code has `msg` type Annotation in TS

### DISCUSSION THREAD

**Discussion by ninesunsabiu** [lingui/js-lingui#1916] [Q&A] [answered]
the code:
```tsx
import { msg } from "@lingui/macro"
import { useLingui } from "@lingui/react";

const title = msg`hi`

type Props = {
    onChange?: (msg: string) => void
}

export const Title = (props: Props) => {
    const { _ } = useLingui()
    return <h1>{_(title)}</h1>
}
```

when exec extract, I got:
<img width="1353" alt="image" src="https://github.com/lingui/js-lingui/assets/19882837/ef05194c-7716-4ba2-99e7-0b42b0a3d77f">

and when I change to:
```diff
 type Props = {
-    onChange?: (msg: string) => void
+    onChange?: (msgs: string) => void
 }
```
it works.

To Reproduce

repro: https://github.com/ninesunsabiu/lingui-brace-bug/blob/main/src/Title.tsx

**Comment by timofei-iatsenko** [comment]
Dug into the issue, and it seems it's a bug in babel-macro-plugin. It's already fixed in `next` branch (we don't relay for the plugin so mush)

**Reply by ninesunsabiu** [reply]
and how `next` branch can fix it? (I switched to the branch, but it still has the error)

**Reply by timofei-iatsenko** [reply, marked as answer]
https://github.com/lingui/js-lingui/pull/1867
